ggshield/cmd/status.py

```diff
@@ -29,8 +30,12 @@ def status_cmd(ctx: click.Context, **kwargs: Any) -> int:
     if not isinstance(response, HealthCheckResponse):
         raise UnexpectedError("Unexpected health check response")
 
+    # Add the API URL to the JSON output
+    json_output = response.to_dict()
+    json_output["instance"] = client.base_uri
+
     click.echo(
-        response.to_json()
+        json.dumps(json_output, indent=2)
```

Comment:
It would maybe be better to change `HealthCheckResponse` and `health_check` to keep `response.to_json()` here.

Reply:
`HealthCheckResponse` is defined in pygitguardian. We can change it but it requires more work. 
I agree that it would have been more understandable, but I'm not entirely sure of where `HealthCheckResponse` is used. We might need to check the imports before modifying it :)